### PR TITLE
Add trading post name to skins with differing name

### PR DESCRIPTION
**Pull request by Blackgen**
I've added the Item name to the Skin name for Tradingpost prices where they don't match.
This will also show different Items for Buy/Sell price if they differ.

Here is an example:
https://imgur.com/a/uB3ZJCa

It works in both views: Analyse and the browsing view

### CODE REVIEW

File: src/main/java/au/net/immortius/wardrobe/site/entities/SameSkinEntry.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package au.net.immortius.wardrobe.site.entities;
+
+public class SameSkinEntry {
```
Comment (immortius) [inline src/main/java/au/net/immortius/wardrobe/site/entities/SameSkinEntry.java]: Doesn't look like this is used anywhere?

File: src/main/java/au/net/immortius/wardrobe/site/GenerateContent.java [hunk 2]
```diff
@@ -36,23 +36,22 @@
...
+    private static final GenericType<Map<Integer, TradingPostEntry>> PRICE_MAP_TYPE = new GenericType<Map<Integer, TradingPostEntry>>() {
     };
     private static final GenericType<Map<Integer, Collection<Integer>>> UNLOCK_ITEM_MULTIMAP = new GenericType<Map<Integer, Collection<Integer>>>() {
     };
     private static final GenericType<List<VendorData>> VENDOR_DATA_LIST_TYPE = new GenericType<List<VendorData>>() {
     };
     private static final GenericType<Set<Integer>> INTEGER_SET_TYPE = new GenericType<Set<Integer>>() {
     };
-
+    private static Logger logger = LoggerFactory.getLogger(GenerateContent.class);
```
Comment (immortius) [inline src/main/java/au/net/immortius/wardrobe/site/GenerateContent.java:54]: Honestly not sure why I didn't make this final like it is supposed to be.

**Review by immortius — COMMENTED** [review]
Other than the one unused (?) class it all looks good to me.

File: src/main/java/au/net/immortius/wardrobe/site/entities/SameSkinEntry.java (hunk 1 above)
Comment (Blackgen) [reply src/main/java/au/net/immortius/wardrobe/site/entities/SameSkinEntry.java]: Jup this seems to be dead code from a previous implementation. I'll remove it

**Pull request merged**